components/userDashboard/application.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import TabsWithTable from "../../components/basic/TabsWithTable";
+
+const Application = ({ user }) => {
+  const [applications, setApplications] = useState([]);
+  const [error, setError] = useState("");
+  const userEmail = user.email;
+
+  const fetchApplications = async () => {
+    try {
+      setError("");
+      const response = await axios.get(
+        `${process.env.NEXT_PUBLIC_BACKEND_URL}/api/jobApplication/email/${userEmail}`,
+        {
+          withCredentials: true,
+        },
+      );
+      setApplications(response.data || []);
+    } catch (error) {
+      setError(error.response?.data?.message || "An error occurred");
+    }
+  };
+
+  useEffect(() => {
+    fetchApplications();
+  }, [userEmail]);
+
+  const formatDate = (isoDate) => {
+    if (!isoDate) return "...";
+    const date = new Date(isoDate);
+    const day = date.getDate();
+    const month = date.toLocaleString("default", { month: "short" });
+    const year = date.getFullYear();
+    return `${day} ${month} ${year}`;
+  };
+
+  const filterFunction = (application, activeTab) => {
+    if (activeTab === "Active") {
+      return (
+        application[4] === "under review" || application[4] === "In Process"
```

Comment:
can we destructure application data and then use? without using `application[4]`. as other engineers will not get.